pkg/app/run_test.go

```diff
@@ -259,7 +259,11 @@ SUBCOMMANDS
   service list [<flags>]
     List Fastly services
 
-    -j, --json  Render output as JSON
+        --direction=ascend   Direction in which to sort results
```

Comment:
It's quite a journey to get to the possible options for this param - you have to go to the 'SEE ALSO' URL then find the API equiv link at the bottom of the page... should we copy the values here?  It looks like we're baking these values into the CLI source anyway: https://github.com/fastly/cli/pull/511/files#diff-df2386696f2988b6e5558b1bf966261e62f37503d021a459ba60dfbe59b37d55R16, so if they were changed upstream in the API, we'd be looking at making CLI updates.

Reply:
@triblondon 

re:

> It's quite a journey to get to the possible options for this param

We implement this flag with an enum and provide hints to the accepted values, so when typing `--direction` if you press `<Tab>` you'll see the values you're allowed to use...

<img width="639" alt="Screenshot 2022-01-18 at 11 17 56" src="https://user-images.githubusercontent.com/180050/149927720-f6b91dd7-ba90-4673-9cf7-52ace50dfc64.png">

Reply:
@triblondon I'm also not sure how easy it would be to pass per-command information within the `SEE_ALSO` section of the help. Currently the information we display is easily generated because we have a common pattern between the information exposed by the go argument parser library and the online URL. It's not impossible to implement a solution, but it's out of scope of this PR.

Reply:
@triblondon And yes you are correct that if the API changes the values it accepts, then the CLI would require an update.